server/php-engine.cpp

```diff
@@ -1840,12 +1841,23 @@ int main_args_handler(int i, const char *long_option) {
       return 0;
     }
     case 's': {
```

Comment:
Also let's add warning in sentry about deprecated option here with this macro `OPTION_ADD_DEPRECATION_MESSAGE('-s/--cluster-name')`